docs/cudf/source/developer_guide/contributing_guide.md

```diff
@@ -0,0 +1,120 @@
+# Contributing Guide
+
+This document focuses on a high-level overview of best practices in cuDF.
+
+## Directory structure and file naming
+
+cuDF generally presents the same importable modules and subpackages as pandas.
+All Cython code is contained in `python/cudf/cudf/_lib`.
+
+**Open question**: Should we start enforcing a stricter file/directory layout? Any suggestions?
+
+
```

Comment:
Wondering if we should mention https://docs.rapids.ai/contributing for Issue, PR basics?

Reply:
No, I think if that belongs anywhere it is in the top-level CONTRIBUTING.md.